lib/Sema/DebuggerTestingTransform.cpp

```diff
@@ -33,35 +34,6 @@ using namespace swift;
 
 namespace {
 
-/// Find available closure discriminators.
-///
-/// The parser typically takes care of assigning unique discriminators to
-/// closures, but the parser is unavailable to this transform.
-class DiscriminatorFinder : public ASTWalker {
```

Comment:
Note: I moved `DiscriminatorFinder` to a shared location `lib/Sema/CodeSynthesis.h`.